src/packages/sdk/src/__tests__/__snapshots__/convertCredentials.test.ts.snap

```diff
@@ -311,23 +347,25 @@ Object {
 
 exports[`Convert postgresql://other@localhost/otherdb?schema=my_schema&connect_timeout=10&application_name=myapp 2`] = `"postgresql://other@localhost/otherdb?schema=my_schema&connect_timeout=10&application_name=myapp"`;
 
-exports[`Convert postgresql://root:prisma@/prisma?host=/var/run/postgresql/ 1`] = `
+exports[`Convert postgresql://root:prisma@localhost/prisma?host=/var/run/postgresql/ 1`] = `
 Object {
   "database": "prisma",
-  "extraFields": Object {},
-  "host": undefined,
+  "extraFields": Object {
+    "host": "/var/run/postgresql/",
+  },
+  "host": "localhost",
   "password": "prisma",
   "port": undefined,
   "schema": "public",
-  "socket": "/var/run/postgresql/",
+  "socket": undefined,
   "ssl": false,
   "type": "postgresql",
-  "uri": "postgresql://root:prisma@/prisma?host=/var/run/postgresql/",
+  "uri": "postgresql://root:prisma@localhost/prisma?host=/var/run/postgresql/",
```

Comment:
The downside of this PR is that this URL now errors like (hostname is required)

```
> new URL('postgresql://root:prisma@/prisma?host=/var/run/postgresql/')
Uncaught:
TypeError [ERR_INVALID_URL]: Invalid URL: postgresql://root:prisma@/prisma?host=/var/run/postgresql/
    at onParseError (internal/url.js:258:9)
    at new URL (internal/url.js:334:5)
    at REPL17:1:1
    at Script.runInThisContext (vm.js:132:18)
    at REPLServer.defaultEval (repl.js:484:29)
    at bound (domain.js:430:14)
    at REPLServer.runBound [as eval] (domain.js:443:12)
    at REPLServer.onLine (repl.js:817:10)
    at REPLServer.emit (events.js:327:22)
    at REPLServer.EventEmitter.emit (domain.js:486:12) {
  input: 'postgresql://root:prisma@/prisma?host=/var/run/postgresql/',
  code: 'ERR_INVALID_URL'
}
```

Reply:
This error is now caught and replaced with in uriToCredentials function
`Invalid data source URL. See https://www.prisma.io/docs/reference/database-reference/connection-urls`